openlibrary/i18n/es/messages.po

```diff
@@ -3701,37 +3309,37 @@ msgid "who have written the most books on this subject"
 msgstr "quienes han escrito más libros sobre este tema"
 
 #: openlibrary/templates/widget.html:33
-#, fuzzy, python-format
+#, python-format
 msgid "Read \"%(title)s\""
-msgstr "Editar %(title)s"
+msgstr "Leer \"%(title)s\""
 
 #: openlibrary/templates/widget.html:38
-#, fuzzy, python-format
+#, python-format
 msgid "Borrow \"%(title)s\""
-msgstr "Editar %(title)s"
+msgstr "Pedir prestado \"%(title)s\""
 
 #: openlibrary/templates/widget.html:44
 #, python-format
 msgid "Join waitlist for &quot;%(title)s&quot;"
-msgstr ""
+msgstr "Unirse a la lista de espera para &quot;%(title)s&quot;"
```

Comment:
Should this use << >> instead of quotes?

Reply:
I'm not sure if I understood the question. I think it is better to use quotes ("...") in all cases.

The correct thing in Spanish is to use Latin quotes («...»). The Spanish Language Academy (RAE) recommends using Latin quotes («...»), and reserve double quotes ("...") for text already put in quotation marks (e.g. «This text contains "quotes" inside»).

However, Spanish-speaking people usually use quotes ("...") when writing or reading on a computer.  I usually use Latin quotes when I write, but I think it might be a good idea to use only the most common quotes ("...") in all cases to avoid mixing two different types of quotation marks in the translation. Spanish speaking people are very used to them.

Reply:
P.S.: In any case, feel free to use the quotation marks you consider. Both double quotes and Latin quotes will work fine.

Reply:
```suggestion
msgstr "Unirse a la lista de espera para «%(title)s»"
```